vfs-azure/src/main/java/com/sludev/commons/vfs2/provider/azure/AzFileObject.java

```diff
@@ -71,122 +68,101 @@
 public class AzFileObject extends AbstractFileObject {
 
     private static final Logger log = LoggerFactory.getLogger(AzFileObject.class);
-
-    private final AzFileSystem fileSystem;
-    private CloudBlobContainer currContainer;
-    private CloudBlockBlob currBlob;
-    private BlobProperties currBlobProperties;
-
     private static final int MEGABYTES_TO_BYTES_MULTIPLIER = (int) Math.pow(2.0, 20.0);
-
+    public static final int BLOB_COPY_THRESHOLD = 256 * 1024 * 1024;
+    static Integer UPLOAD_BLOCK_SIZE = 20; //in MB's
     private static Boolean ENABLE_AZURE_STORAGE_LOG = false;
 
-    static Integer UPLOAD_BLOCK_SIZE = 3; //in MB's
 
-    private static Tika tika = new Tika();
+    private FileType fileType = null;
 
     static {
 
         String uploadBlockSizeProperty = System.getProperty("azure.upload.block.size");
-        UPLOAD_BLOCK_SIZE = (int) NumberUtils.toLong(uploadBlockSizeProperty, UPLOAD_BLOCK_SIZE) * MEGABYTES_TO_BYTES_MULTIPLIER;
+
+        UPLOAD_BLOCK_SIZE = (int) NumberUtils.toLong(uploadBlockSizeProperty, UPLOAD_BLOCK_SIZE); //*
+        // MEGABYTES_TO_BYTES_MULTIPLIER;
```

Comment:
Is this deliberately commented out?
* if not, whoops, please fix;
* if so, then:
** why not remove it entirely?
** the info log about 15 lines below is rendered incorrect, as it says "Bytes" when referring to this value.

Reply:
Looks to me like `uploadBlockSizeProperty` is no longer used at all.  I suggest you remove these few lines of code, and delete all reference to the `azure.upload.block.size` property from the repo.